bpf/include/bpf/ctx/xdp.h

```diff
@@ -270,31 +272,33 @@ ctx_full_len(const struct xdp_md *ctx)
 	return ctx_data_end(ctx) - ctx_data(ctx);
 }
 
+struct bpf_elf_map __section_maps cilium_xdp_scratch = {
+	.type		= BPF_MAP_TYPE_PERCPU_ARRAY,
+	.size_key	= sizeof(int),
+	.size_value	= META_PIVOT,
+	.pinning	= PIN_GLOBAL_NS,
+	.max_elem	= 1,
+};
```

Comment:
Should we open or create this map from cilium-agent (cf. #9276)?

Reply:
> Should we open or create this map from cilium-agent (cf. #9276)?

Why would it be needed?

Reply:
It's not needed. I think the goal was just to ensure we keep track of all global maps on Go side and have already opened all maps when we load BPF programs.
It seems less important for maps we don't use in `bpf_lxc` so I'm okay if we don't do that for this map.